core/src/com/softwarearchitecture/ECS/Card.java

```diff
@@ -0,0 +1,5 @@
+package com.softwarearchitecture.ecs;
+
+public interface Card {
```

Comment:
This is a very important concept in our application and as it is an empty interface it should have javadocs to describe what this is